### PR TITLE
duplicate quote and typo

### LINKED ISSUE #5346: Dupplicated quote
Hi,
I noticed that in https://github.com/notepad-plus-plus/notepad-plus-plus/blob/3546268c235780a6ba15266e7685aab7f6d6ce7f/PowerEditor/src/Notepad_plus.cpp the quote Anonymous 105 (line 6255) is the same as the quote Anonymous 151 (line 6301)

### PR DESCRIPTION
**Pull request by SinghRajenM**
Fixed #5346

### CODE REVIEW

File: PowerEditor/src/Notepad_plus.cpp [hunk 1]
```diff
@@ -6250,7 +6250,7 @@ static const QuoteParams quotes[] =
 	{TEXT("Anonymous #101"), QuoteParams::rapid, true, SC_CP_UTF8, L_TEXT, TEXT("It's always sad when a man and his dick share only one brain...\nand it turns out to be the dick's.")},
 	{TEXT("Anonymous #102"), QuoteParams::rapid, true, SC_CP_UTF8, L_TEXT, TEXT("If IE is brave enough to ask you to set it as your default browser,\ndon't tell me you dare not ask a girl out.")},
 	{TEXT("Anonymous #104"), QuoteParams::rapid, true, SC_CP_UTF8, L_TEXT, TEXT("The main idea of \"Inception\":\nif you run a VM inside a VM inside a VM inside a VM inside a VM,\neverything will be very slow.")},
-	{TEXT("Anonymous #105"), QuoteParams::rapid, true, SC_CP_UTF8, L_TEXT, TEXT("Q: What's the object-oriented way to become wealthy?\nA: Inheritance.")},
+	//{TEXT("Anonymous #105"), QuoteParams::rapid, true, SC_CP_UTF8, L_TEXT, TEXT("Q: What's the object-oriented way to become wealthy?\nA: Inheritance.")}, Duplicate of 105
```
Comment (donho) [inline PowerEditor/src/Notepad_plus.cpp]: Please use the following instead:
	//{TEXT("Anonymous #105"), QuoteParams::rapid, true, SC_CP_UTF8, L_TEXT, TEXT("")},

Comment (SinghRajenM) [reply PowerEditor/src/Notepad_plus.cpp]: Done!

Comment (SinghRajenM) [reply PowerEditor/src/Notepad_plus.cpp]: @donho looks like resolving conversation does not send notification unless it is marked with symbol '@'. 

Review changes are accommodated.

**Comment by donho** [comment]
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/1af4d0f7f4b50b016e47b31fc9264f389045445b

**Pull request closed without merging**